packages/passport/sdk/src/zkEvm/walletHelpers.test.ts

```diff
@@ -53,4 +56,57 @@ describe('getSignedTypedData', () => {
 
     expect(signature).toEqual(expectedSignature);
   });
+
+  describe('when the EOA address is smaller than the relayer address', () => {
+    it('should include the EOA signature in the combined signature first', async () => {
+      // The following wallet has an address of `0x15...` which is SMALLER than the relayer address (`0x1B...`),
```

Comment:
nit; it will be the `ImmutableSigner` contract address

Reply:
Fixed ✅